package.json

```
@@ -12,7 +12,9 @@
     "circularity": "madge --circular packages/*/src",
     "generate:component": "davinci-code new component",
     "generate:example": "davinci-code new example",
-    "generate:icons": "./bin/generate-icons",
+    "generate:components-from-svg": "yarn generate:icons && yarn generate:pictograms",
```

```suggestion
    "generate:svg-components": "yarn generate:icons && yarn generate:pictograms",
```
Just an idea / small nit: sometimes you don't have autocomplete and typing a long command is more painful :smile:

---

@mkrl Sounds good, added!